ampbench_lib.js

```diff
@@ -775,10 +775,14 @@ function lib_fetch_cdn_validator_spec_file_revision(callback) {
  */
 function lib_load_validator(opt_force_reload) {
     let _force_reload = opt_force_reload || false;
-    const _validator = amphtml_validator.getInstanceFromFileSync(VALIDATOR_JS_FILE, _force_reload);
-    amphtml_validator_instance = _validator; // cache the module level instance here
-    amphtml_validator_spec_file_revision = lib_extract_validator_spec_file_revision(_validator.scriptContents);
```

Comment:
Yes I think a content checksum is better for distinguishing validator releases.
I think if the specfile revision is useful, we should provide some programmatic access to it, as in, not something via grepping throught the contents of the Javascript file. To keep with the structure of how we have it working in C++, we could put it into the ValidationResult, since the .proto already has such a field. Or we could make some method that returns it. Either way, I think it should be a different mechanism than the change detection based on the contents.